testdata/goldens/java/overview.html

```diff
@@ -1,18 +1,18 @@
-﻿<!DOCTYPE html>
-<html devsite="">
-  <head>
-    <meta name="project_path" value="/java/docs/reference/_project.yaml">
-    <meta name="book_path" value="/java/docs/reference/google-cloud-speech/latest/_book.yaml">
-  </head>
-  <body>
-    {% verbatim %}
-    <div>
-      <article data-uid="google-cloud-speech">
-<h1 class="page-title">google-cloud-speech overview (2.0.0)</h1>
-  
-    <h2 id="packages">
-  </h2>
-      <h3><a class="xref" href="com.google.cloud.speech.v1.html">com.google.cloud.speech.v1</a></h3>
+﻿<!DOCTYPE html>
+<html devsite="">
+  <head>
+    <meta name="project_path" value="/java/docs/reference/_project.yaml">
+    <meta name="book_path" value="/java/docs/reference/google-cloud-speech/latest/_book.yaml">
+  </head>
+  <body>
+    {% verbatim %}
+    <div>
+      <article data-uid="google-cloud-speech">
+<h1 class="page-title">google-cloud-speech overview (2.0.0)</h1>
+  
+    <h2 id="packages">
+  </h2>
+      <h2><a class="xref" href="com.google.cloud.speech.v1.html">com.google.cloud.speech.v1</a></h2>
```

Comment:
`com.google.cloud.speech.v1` is a package, so do we know what the above and empty `<h2 id="packages">` is meant to be used for?